chain/txValidator.go

```diff
@@ -188,6 +187,9 @@ func VerifyTransactionWithLedger(txn *transaction.Transaction) error {
 		return errors.New("Unpack transactiion's paylaod error")
 	}
 
+	sender, _ := ToCodeHash(txn.Programs[0].Code)
```

Comment:
Coinbase txn does not have programs, so this will cause panic

Reply:
We should also try to avoid omitting error using "_"

Reply:
done